PAPP 30722 : Streamline Permissions (#12)
splunk-soar-connectors/microsoftteams: manual_readme_content.md

    file)
-   File owner: Appropriate phantom user

## Configure the Microsoft Teams Splunk SOAR app asset

When creating an asset for the **Microsoft Teams** app, place the **Application Id** of the app in
the **Client ID** field and place the password generated during the app creation process in the
**Client Secret** field. Then, after filling out the **Tenant ID** field, click **SAVE** . Both the
Application/Client ID and the Tenant ID can be found in the **Overview** tab on your app's Azure
page.  

> Added a new **Scope** configuration parameter with minimum required scopes as the default value, and you can add/delete the scopes from the value according to the requirement. The scopes added in this parameter should be same as added while creating the application on azure portal

Reviewer: ```suggestion
The **Scope** configuration parameter's default value is the minimum required scopes. You can add/delete scopes as needed. The scopes added in this parameter should be consistent with those used to create the application on the Azure portal
```
